### ISSUE TITLE
Add rk3318 Support

### ISSUE THREAD

**Issue by sajati** [unifreq/openwrt_packit#128]
Hi, i just copied the rk3328 files and mk script and added **idbloader.img** and **u-boot.itb** from armbian, it works on my H96max rk3318 (black pcb)
but the sound isn't working yet, since i notice about sound in mk script and files, i assume you did make the sound work on rk3328, would you mind assist me to make the sound work ? either by RCA port or HDMI port, and when i open **snd-rk3328** file it just contains some list, perhaps possible getting correct values from running armbian ?

**Comment by unifreq** [comment]
Yes, these are the relevant sound card kernel modules list. These kernel modules will be loaded, assuming that the kernel driver exists as a module. In armbian, you can use lsmod to see which sound modules are automatically loaded.

**Comment by sajati** [comment]
Thanks @unifreq , my system running so stable, except wifi, from what i read from internet my board use broadcom 4334, which isnt exist in `lib/firmware/brcm`, i think im going to build the kernel using your source https://github.com/unifreq/linux-5.19.y
but i have no knowledge on doing it, is i just need to clone it and run `make` ?
or perhaps run `make` with some variable exported first ?
my linux environment is ubuntu jammy under windows 10 wsl
